Tune README files with more info (#106)
imrehg/cdei-development: notebooks/README.md

# ML Bias notebooks

This directory contains the notebooks referenced in write-up on the [main website][main-site].

> To start experiment with these notebooks, open [them online using Binder][binder-notebooks]. Binder which allows you to interact with these notbooks in a cloud-based [Jupyter][jupyter] environment.

Reviewer: ```suggestion
To start an experiment with these notebooks, [open them online on Binder][binder-notebooks]. Binder allows you to interact with these notebooks in a cloud-based [Jupyter][jupyter] environment.
```
